spring-batch-infrastructure/src/main/java/org/springframework/batch/item/data/AbstractNeo4jItemReader.java

```diff
@@ -184,9 +184,7 @@ protected String generateLimitCypherQuery() {
 
 		String resultingQuery = query.toString();
 
-		if (logger.isDebugEnabled()) {
-			logger.debug(resultingQuery);
-		}
+		logger.debug(resultingQuery);
```

Comment:
This seems an unintentional change (as it goes against the idea of the PR). I will revert it on merge.